src/wily/commands/diff.py

```diff
@@ -134,23 +134,19 @@ def diff(config, files, metrics, changes_only=True, detail=True, revision=None):
             if metric.type in (int, float) and new != "-" and current != "-":
                 if current > new:
                     metrics_data.append(
-                        "{0:n} -> \u001b[{2}m{1:n}\u001b[0m".format(
-                            current, new, BAD_COLORS[metric.measure]
-                        )
+                        f"{current:n} -> [{BAD_COLORS[metric.measure]}m{new:n}[0m"
                     )
                 elif current < new:
                     metrics_data.append(
-                        "{0:n} -> \u001b[{2}m{1:n}\u001b[0m".format(
-                            current, new, GOOD_COLORS[metric.measure]
-                        )
+                        f"{current:n} -> [{GOOD_COLORS[metric.measure]}m{new:n}[0m"
```

Comment:
Please verify that you've run this `wily diff` on a project and check that the colors are rendering correctly. In GitHub diff this looks like an invalid character